code/__HELPERS/atoms.dm

```diff
@@ -313,3 +313,17 @@ rough example of the "cone" made by the 3 dirs checked
 ///A do nothing proc
 /proc/pass(...)
 	return
+
+///Returns a list of the parents of all storage components that contain the target item
+/proc/get_storage_locs(obj/item/target)
+	. = list()
+	if(!target || !istype(target) || !(target.item_flags & IN_STORAGE))
```

Comment:
``!istarget || istype(target)`` is redundent and should just be ``!istype(target)``

but then again I think that both should be removed because if get_storage_locs(null) is called, it should runtime to tell coders that something is wrong.

Reply:
related helpers choose to early-return over runtiming, which I think is what maintainers prefer.